packages/cysync-core/src/dialogs/Send/context/index.tsx

```diff
@@ -60,50 +95,315 @@ export const SendDialogProvider: FC<SendDialogContextProviderProps> = ({
 }) => {
   const lang = useAppSelector(selectLanguage);
   const dispatch = useAppDispatch();
+  const { priceInfos } = useAppSelector(selectPriceInfos);
   const deviceRequiredDialogsMap: Record<number, number[] | undefined> = {
-    1: [0],
+    3: [0],
+    4: [0],
+    5: [0],
   };
```

Comment:
I think we only require device connection at 3rd tab, right?

Reply:
We require 3rd and 4th, updated.